src/hooks/accounting.ts

```diff
@@ -93,9 +91,10 @@ export const useAccounting = (): UseAccountingHook => {
     if (isLoadingUncashed || !settlements.settlements || uncashedAmounts || error) return
 
     setIsloadingUncashed(true)
-    const promises = settlements.settlements.settlements.map(({ peer }) =>
-      beeDebugApi.chequebook.getPeerLastCashout(peer),
-    )
+    const promises = settlements.settlements.settlements
```

Comment:
This looks interesting 🤔

Reply:
Yeah this should be extracted into a provider and we can drop at least one of these. Not a part of this PR, but there is an issue for it and I want to address it soon #40